lib/ansible/modules/identity/ipa/ipa_user.py

```diff
@@ -250,6 +265,7 @@ def main():
                          mail=dict(type='list'),
                          sn=dict(type='str'),
                          uid=dict(type='str', required=True, aliases=['name']),
+                         gidnumber=dict(type='str'), uidnumber=dict(type='str'),
```

Comment:
would you mind creating 2 lines here? for each param?

Reply:
done